app-drawer/app-drawer.html

```diff
@@ -45,8 +45,10 @@
 `--app-drawer-content-container` | Mixin for the drawer content container | {}
 `--app-drawer-scrim-background`  | Background for the scrim               | rgba(0, 0, 0, 0.5)
 
-@demo app-drawer/demo/demo1.html
-@demo app-drawer/demo/demo2.html
+@group App Elements
+@element app-drawer
+@demo app-drawer/demo/index.html Simple Left Drawer
```

Comment:
Do we have a sample of this index page? I just saw that iron-list had one index.html and a bunch of other samples https://github.com/PolymerElements/iron-list/blob/master/iron-list.html#L142-L149